_pages/en_US/godmode9-usage.txt

```diff
@@ -122,18 +122,52 @@ Insert the game cartridge you intend to dump into your device
   + **NDS Game Cartridge:** Press (A) on `[TitleID].trim.nds` to select it, then select "Copy to 0:/gm9/out"
 1. Your installable `.cia` or non-installable `.nds` formatted file will be outputted to the `/gm9/out/` folder on your SD card
 
-## Dumping a Title
+## Dumping a Game Cartridge's contents
+
+1. Insert a game cartridge you wish to dump from
+1. Press and hold (Start), and while holding (Start), power on your device. This will launch GodMode9
+1. Navigate to `[C:] GAMECART`
+1. Hover over `[TitleID].trim.3ds` and press (A)
+1. Select "NCCH image options"
+1. Select "Mount image to drive"
+1. Press (A) to continue
+1. Hover over the `exefs` directory, hold (R) and press (A) to open the directory options
+1. Select "copy to 0:/gm9/out", do the same with the `romfs` directory
+  + The dumped files will be in `/gm9/out` on the sd card
```

Comment:
Capitalize "sd", also I checked other parts of the guide and they all seem to use "in the /whatever/whatever folder on your SD card" so maybe you could change it to that